dolphin/alert_manager/rpcapi.py

```diff
@@ -35,7 +35,7 @@ class AlertAPI(object):
 
     def __init__(self):
         super(AlertAPI, self).__init__()
-        target = messaging.Target(topic=CONF.snmp_config_topic,
+        target = messaging.Target(topic=CONF.dolphin_alert_topic,
```

Comment:
suggestion: not to use project name in variables or config options.

Reply:
project name is a namespace in rabbitmq to avoid topic conflict.